<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bfsign</class>
 <widget class="QWidget" name="bfsign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QFrame{
border:sold 10px rgba(0,0,0);
background-image: url(:/bg/image/bg3.jpg);
}
QWidget{
border:sold 10px rgba(0,0,0);
	

}
QLineEdit{
color:#8d98a1;
background-color:#405361;
font-size:16px;
border-style:inset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:#ced1d8;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QPushButton:pressed{
background-color:rgb(224,0,0);
border-style:inset;
font-style:MingLiU-ExtB;
}

QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入用户信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>新的用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_key">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>新的密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_akey">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>再次输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>133</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-style:rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>管理员验证码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
